versioned_docs/version-1.0/tutorial/01-first-steps.md

```diff
@@ -24,14 +24,19 @@ Please see the [Cadence introduction](../index.md) for more information about th
 
 ---
 
-The [Flow Playground](https://play.onflow.org) includes
-an in-browser editor and emulator to experiment with Flow.
+The [Flow Playground](https://play.flow.com) includes
+an in-browser editor and Flow emulator to experiment with Flow.
 Using the Flow Playground, you can write Cadence smart contracts,
 deploy them to a local Flow emulated blockchain, and submit transactions.
 
 The Flow Playground should be compatible with any standard web browser,
 but we recommend that you use Google Chrome with it,
-because it has been tested and optimized for only the Chrome browser so far.
+because it has been tested and optimized primarily with the Chrome browser so far.
```

Comment:
The Flow Playground should work with any standard web browser. However, we recommend using Google Chrome, as it has been primarily tested and optimized for this browser.